docs/pages/installation/aks.md

```diff
@@ -8,70 +8,68 @@ language: en
 list: include
 image: /assets/img/platforms/aks.svg
 ---
-Use Meshery to manage your AKS clusters. Deploy Meshery on AKS or outside of AKS.
-
+Manage your AKS clusters with Meshery. Deploy Meshery on AKS or outside of AKS.
 {% include installation_prerequisites.html %}
 
-#### Prequisites:
-1. Access to an active AKS cluster in one of your resource groups. 
-2. Azure CLI `az` installed on your machine.
+### General Prerequisites:
+1. Access to an active AKS cluster in one of your resource groups.
+2. Ensure you have the Azure CLI `az` installed on your machine.
+
+### Install Meshery on an AKS Cluster using mesheryctl
 
-## Manage AKS clusters with Meshery
+## Connect to Azure Kubernetes Services Cluster using mesheryctl
 
-- [Connect Meshery to your AKS cluster](#automatically-connect-meshery-to-azure-kubernetes-cluster)
+Use Meshery's CLI to streamline your connection to your AKS cluster. Configure Meshery to connect to your AKS cluster by executing:
 
-### Automatically connect Meshery to your Azure Kubernetes Cluster
+{% capture code_content %}$ mesheryctl system config aks{% endcapture %}
```

Comment:
Sometimes, we face issues with the token not being present in the default path. Do we want to add any steps on how to get over such issues, i.e. calling `mesheryctl system login` and click GetToken for fetching token.json file

Reply:
yeah sure is the method mention in any other docs ? so i can replicate it or make that a seperate md file and make a hyperlink over here

Reply:
@hexxdump added a hyperlink to resolve authentication issue in the last commit https://github.com/meshery/meshery/pull/9077/commits/e8c283e89817628a49ef284407f331d55c0bbb14